lib/lib_rss.php

```diff
@@ -548,15 +548,27 @@ function getNonStandardShortcuts($shortcuts) {
 }
 
 function errorMessage($errorTitle, $error = '') {
-	// Prevent empty <h2> tags by checking if error isn't empty first
-	if ('' !== $error) {
+	$errorTitle = htmlspecialchars($errorTitle, ENT_NOQUOTES, 'UTF-8');
+
+	$message = '';
+	$details = '';
+	// Prevent empty tags by checking if error isn not empty first
+	if ($error) {
 		$error = htmlspecialchars($error, ENT_NOQUOTES, 'UTF-8');
-		$error = "<h2>{$error}</h2>";
+
+		// First line is the main message
+		$message = strtok($error, "\n");
+		// Other lines are the details
+		$details = strtok('');
```

Comment:
I didn't know about `strtok`, it has a very strange behaviour… I think `list($message, $details) = explode("\n", $error, 2)` is the same and a bit more clear (?)

Reply:
I do not mind. I do not think it is more clear, but rather might be a question of habits :-)

Reply:
Changed https://github.com/FreshRSS/FreshRSS/pull/3589/commits/ee0c8e872134ea81ca46d4f629ae83521de296d7